README.md

```diff
@@ -13,7 +13,7 @@
 
 <!-- markdownlint-enable -->
 
-# iceoryx2 - Zero-Copy Lock-Free IPC Purely Written In Rust
+# iceoryx2 - Zero-Copy Lock-Free IPC with a Rust Core
```

Comment:
I would keep it with purely written in rust. It has language bindings but there is actually no logic or algorithm of the inter-process communication written in anything else then rust.

Reply:
@elfenpiff I removed it because I thought it might be confusing in the Python Package.

Also purely implies there are no other langauges, which is not correct.